charts/umbrella/values.yaml

```diff
@@ -681,6 +681,149 @@ managed-identity-wallet:
       nginx.ingress.kubernetes.io/cors-allow-methods: GET
       nginx.ingress.kubernetes.io/enable-cors: "true"
 
+bpdm:
+  enabled: false
+  postgres:
+    enabled: true
+    architecture: standalone
+    primary:
+      persistence:
+        enabled: false
+    fullnameOverride: &bpdm-postgres-host bpdm-postgres
+    auth:
+      database: bpdm
+      username: bpdm
+      # Change for production
+      password: bpdm
+  keycloak:
+    enabled: false
+  bpdm-gate:
+    enabled: true
+    postgres:
+      enabled: false
+    ingress:
+      enabled: true
+      annotations:
+        kubernetes.io/ingress.class: nginx
+        nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+      hosts:
+        - host: partners-gate.tx.test
+          paths:
+            - path: /
+              pathType: Prefix
+    springProfiles:
```

Comment:
not needed anymore, the default configuration already enbales auth. see: https://github.com/eclipse-tractusx/bpdm/blob/release/6.0.x/INSTALL.md